Source/Core/ElementUtilities.cpp

```diff
@@ -153,6 +153,7 @@ int ElementUtilities::GetLineHeight(Element* element)
 	ROCKET_UNUSED_SWITCH_ENUM(Property::ABSOLUTE_UNIT);
 	ROCKET_UNUSED_SWITCH_ENUM(Property::PPI_UNIT);
 	ROCKET_UNUSED_SWITCH_ENUM(Property::RELATIVE_UNIT);
+	ROCKET_UNUSED_SWITCH_ENUM(Property::REM);
```

Comment:
How can you be sure this never happens? Maybe it's supposed to hit the return 0 after the switch.

Reply:
AFAIK libRocket uses CSS2 and that didn't support `rem` for `line-height` so the idea is to ignore it and return 0.

Reply:
That doesn't ignore it though, it throws an assertion.

Reply:
I would say even better given that's not supported!
Also consider that `rem` has been added later to libRocket (it wasn't part of the original development since it's not part of CSS2) and in CSS3 its use for `line-height` is strongly discouraged (see https://stackoverflow.com/a/21380378).

Reply:
Sounds good.

Reply:
Question: if one day we want to read some page from the web like warsow did in the past, basically by downloading a simplified (because of smaller HTML/CSS subset of libRocket) version of blog post and other news, what is happening in case of assertion?

Reply:
In the production release, nothing.

Reply:
So I have nothing against that.